app/views/forums/posts/_post.html.haml

```diff
@@ -8,7 +8,9 @@
         = link_to post_p.edit_path, class: 'btn btn-link btn-sm' do
           %span.glyphicon.glyphicon-cog
     .pull-right
-      = post_p.created_at
+      = link_to "#post_#{post.id}" do
+        %span.glyphicon.glyphicon-link.small
+        post_p.created_at.to_s
```

Comment:
Hmm, weird. With the current code I have there it works for me locally, I'll give it a look tomorrow.